<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sim_Window</class>
 <widget class="QMainWindow" name="Sim_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simulator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="Render_Widget" name="render_widget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_simulation"/>
   <addaction name="action_ground"/>
   <addaction name="action_gravity"/>
   <addaction name="separator"/>
   <addaction name="action_reset"/>
   <addaction name="action_stop_motion"/>
   <addaction name="separator"/>
   <addaction name="action_show_axis"/>
   <addaction name="action_show_altitude"/>
  </widget>
  <action name="action_simulation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/simulator.png</normaloff>:/icons/simulator.png</iconset>
   </property>
   <property name="text">
    <string>Simulation</string>
   </property>
  </action>
  <action name="action_reset">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/reconnect.png</normaloff>:/icons/reconnect.png</iconset>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="action_stop_motion">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/stop.png</normaloff>:/icons/stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop Motion</string>
   </property>
  </action>
  <action name="action_ground">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/ground.png</normaloff>:/icons/ground.png</iconset>
   </property>
   <property name="text">
    <string>Ground</string>
   </property>
   <property name="toolTip">
    <string>Ground</string>
   </property>
  </action>
  <action name="action_gravity">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/gravity.png</normaloff>:/icons/gravity.png</iconset>
   </property>
   <property name="text">
    <string>Gravity</string>
   </property>
  </action>
  <action name="action_show_axis">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/Frame.png</normaloff>:/icons/Frame.png</iconset>
   </property>
   <property name="text">
    <string>Show Axis</string>
   </property>
  </action>
  <action name="action_show_altitude">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/Distance.png</normaloff>:/icons/Distance.png</iconset>
   </property>
   <property name="text">
    <string>Show Altitude</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Render_Widget</class>
   <extends>QWidget</extends>
   <header>Render_Widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
